File: standing-out/StandingOut.Data/DTO/StripeSubscription.cs
using System;

namespace StandingOut.Data.DTO
{
    public class StripeSubscription
    {
        public StripeSubscription()
        {
        }

        public decimal Amount { get; set; }
        public string Name { get; set; }
        public string Status { get; set; }

        //User For Hide Select button on front end
        public Guid? StripePlanId { get; set; }
    }
}

File: standing-out/StandingOutStore/AngularSource/app/models/stripe-subscription.ts
export class StripeSubscription {
    name: string;
    status: string;
    amount: number;
    stripePlanId: string = null;
}

export class DbscheckApproval {
    adminDashboard_ProfileApproval_ApprovalRequired: boolean;
    adminDashboard_DBSApproval_ApprovalRequired: boolean;
}
